src/full-monero-node-external-sd-install.sh

```diff
@@ -3,8 +3,9 @@
 (
 MONERO=~/monero-cli
 MONERO_CLI=~/monero-cli/monero-cli
-NODE_DATA=~/storage/external-1/bitmonero
-NODE_CONFIG=~/monero-cli/config
+NODE_CONFIG=~/storage/shared/crypto/monero-cli/config
+INTERNAL_NODE=~/storage/shared/crypto/monero-cli/blockchain
```

Comment:
how about `INTERNAL_NODE=~/storage/shared/crypto/monero-cli/bitmonero`

Reply:
Haven uses .haven
Wownero uses .wownero
I'm assuming 
When monero was bitmonero, it used .bitmonero
And that
Monero uses .bitmonero, for backwards compatibility. 

Since we're creating a new folder to store the blockchain, as the default folder is inside of termux app directory, I dont like the pattern

Ie
Monero blockchain folder would be monero

I think "blockchain" is fine.. It's next to "config".
Bitmonero is an old name and "monero" doesn't imply that the blockchain is stored there.